src/main.cpp

```diff
@@ -262,6 +262,7 @@ void setup()
 
 #ifdef DEBUG_PORT
     consoleInit(); // Set serial baud rate and init our mesh console
+    delay(2000);   // TODO: remove me
```

Comment:
@mverch67 - probably want to remove this one?

Reply:
ah, yes... maybe need it only for the Indicator